uploader/templates/upload.html

```diff
@@ -1,389 +1,434 @@
+        .setDeveloperKey(developerKey)
+        .setCallback(pickerCallback)
+        .build();
+      picker.setVisible(true);
+    }
+  }
+
+  function generateGdriveThumbNailLink(itemId) {
+    return `https://drive.google.com/thumbnail?authuser=0&sz=w50-h50&id=${itemId}`;
+  }
+  // A simple callback implementation.
+  function pickerCallback(data) {
+    if (data.action == google.picker.Action.PICKED) {
+      data.docs.forEach(item =>
+        pickedFiles.push({
+          name: item.name,
+          id: item.id,
+          description: item.description
+        })
+      );
+
+      $(".step-3").show();
+      $(".step-1-2").hide();
+
+      fileStagingTable = new Tabulator("#picked-files-table", {
+        data: pickedFiles,
+        layout: "fitColumns",
+        responsiveLayout:"collapse",
+        columns: [
+          {
+            title: "File",
+            field: "id",
+            formatter: function(cell, formatterParams, onRendered) {
+              return `<img src='${generateGdriveThumbNailLink(
+                cell.getValue()
+              )}'>`;
+            },
+            width: 80,
+            responsive:0,
+          },
+          {
+            title: "Title",
+            field: "name",
+            editor: "input",
+            validator: "unique",
+            width: 200,
+            responsive:0,
+          },
+          {
+            title: "Description",
+            field: "description",
+            editor: "textarea",
+            validator: "required",
+            responsive:0,
+            width: 200,
+          },
+          {
+            title: "License",
+            field: "license",
+            width: 200,
+            formatter: function(cell, formatterParams, onRendered) {
+              return `{% include 'licenses.html' %}`;
+            },
+          },
+          {
+            title: "Date of Creation (first published)",
+            field: "date_created",
+            width: 300,
+            formatter: function(cell, formatterParams, onRendered) {
+              return `{% include 'datepicker.html' %}`;
+            },
+            responsive:1, 
+          },            
+          {
+            formatter: "buttonCross",
+            width: 40,
+            align: "center",
+            headerSort: false,
+            cellClick: function(e, cell) {
+              cell.getRow().delete();
+            },
+            responsive:0,
+          }
+        ]
+      });
+      fileStagingTable.setData(pickedFiles);
+    }
+  }
+
+  function uploadFiles() {
+    // Disable the button first.
+    $("#upload-button").prop("disabled", true);
+    $("#upload-button i")
+      .removeClass("fa-caret-right")
+      .addClass("fa-spinner fa-spin");
+    var fileData = {
+      fileList: fileStagingTable.getData(),
+      token: oauthToken
+    };
+
+    for (var x = 0; x < fileData["fileList"].length; x++) {
+      var sell = document.getElementById(`dateCreated`);
```

Comment:
I dont think you can use this. Sadly, there are multiple date inputs with `dateCreated` right now. This would just give one of them. You might want to see how the license thingy works. For example, creating a dynamic id per field. See comment above.